tests/profile/test_views.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+from collections import OrderedDict
+
+from rest_framework.test import APIClient
+from django.urls import reverse
+
+from tests.profile import factoryboy as profile_factoryboy
+from tests.datasets import factoryboy as datasets_factoryboy
+
+@pytest.fixture
+def api_client():
+   return APIClient()
+
+@pytest.fixture
+def profile():
+    return profile_factoryboy.ProfileFactory()
+
+@pytest.fixture
+def indicator_data_items_data():
+    return {
+            "groups": {
+                "age group": {
+                    "20-24": [
+                        {
+                            "count": 2.2397,
+                            "gender": "F"
+                            }
+                        ],
+                    "15-19": [
+                        {
+                            "count": 9.62006,
+                            "gender": "F"
+                            },
+                        {
+                            "count": 8.79722,
+                            "gender": "M"
+                            }
+                        ],
+                    }
+                },
+            "subindicators": { "M": 52.34565, "F": 56.0179 }
+            }
+
+@pytest.fixture
+def expected_age_groups():
+    return OrderedDict([
+           ('15-19',OrderedDict(
+                            M={"count": 8.79722},
+                            F={"count": 9.62006},
+                    )),
+            ('20-24',OrderedDict(F={"count": 2.2397}))
+        ])
+
+@pytest.mark.django_db
+class TestProfileGeographyData:
```

Comment:
why are those tests in your changes? they have nothing to do with your change, right?